11/umbraco-cms/reference/notifications/README.md

```diff
@@ -33,11 +33,14 @@ public void Handle(TemplateSavedNotification notification)
 
 ### Registering notification handlers
 
-Once you've made your notification handlers you need to register them with the `AddNotificationHandler` extension method on the `IUmbracoBuilder`, so they're run whenever a notification they subscribe to is published. There are two ways to do this: In the Startup class, if you're making handlers for your site, or a composer if you're a package developer subscribing to notifications.
+Once you've made your notification handlers, you need to register them with the `AddNotificationHandler` extension method on the `IUmbracoBuilder` to run them whenever a notification they subscribe to is published. There are two ways to do this: 
+
+1. In the **Startup** class, if you're making handlers for your site
+2. In a [**composer**](https://docs.umbraco.com/umbraco-cms/implementation/composing), if you're a package developer subscribing to notifications
```

Comment:
This link here should be a relative one - otherwise it will open in a new tab 😅 

Without checking, it would be something like `../../implementation/composing.md`.

Reply:
Good point, done!